lists/bo/bo-cins

```diff
@@ -0,0 +1,40 @@
+{
+  "name": {
+    "en": "Bolivia Institutional Classifier",
+    "local": "Bolivia Clasificador Institucional"
+  },
+  "url": "https://www.sicoes.gob.bo/portal/clasificadores/institucional.php",
+  "description": {
+    "en": "The institutional classifier reflects the organizational structure of the public sector, taking into account the legal nature, institutional level and entity (code, name and acronym), for the administration of State resources, according to competencies in current regulations."
+  },
+  "coverage": [
+    "BO"
+  ],
+  "structure": [
+    "government_agency"
```

Comment:
National, regional and municipal companies are listed in the drop down under entity types on the site. It might be better to include "company" here and then explain in the textual description.

Reply:
I've included "company" in this array now